src/compiler/diagnosticMessages.json

```diff
@@ -617,8 +617,13 @@
     },
     "Unterminated Unicode escape sequence.": {
         "category": "Error",
-        "code":  1199
+        "code": 1199
     },
+    "Type annotation on export statements are only allowed in ambient module declarations.": {

```

Comment:
A type annotation on an export statement is only allowed in an ambient module declaration.

Btw should it be ambient external module declaration?